experimental/unicodeset_parse/src/parse.rs

```diff
@@ -1099,19 +1107,28 @@ where
         let mut try_sc = Err(PEK::UnknownProperty.into());
         // contains a value for the Script_Extensions property that needs to be tried
         let mut try_scx = Err(PEK::UnknownProperty.into());
+        // contains a value for the Grapheme_Cluster_Break property that needs to be tried
```

Comment:
Nit: Update the comments in this function and the previous function since we now support more than just the ECMA-262 property set.

Reply:
updated